app/src/main/java/woowacourse/movie/activity/ReservationResultActivity.kt

```diff
@@ -20,51 +22,67 @@ class ReservationResultActivity : AppCompatActivity() {
-        if (reservationData != null) {
-            renderMovie(reservationData.movie)
-            renderReservationDetail(reservationData.detail.toDomain())
-        }
+
+        val reservationDetail =
+            intent.extras?.getSerializable<ReservationDetailViewData>(ReservationDetailViewData.RESERVATION_DETAIL_EXTRA_NAME)
+                ?: return finish()
+        val seats = intent.extras?.getSerializable<SeatsViewData>(SeatsViewData.SEATS_EXTRA_NAME)
+            ?: return finish()
+        val price = intent.extras?.getSerializable<PriceViewData>(PriceViewData.PRICE_EXTRA_NAME)
+            ?: return finish()
+        renderReservationDetail(reservationDetail, seats, price)
+
+        val movie = intent.extras?.getSerializable<MovieViewData>(MovieViewData.MOVIE_EXTRA_NAME)
+            ?: return finish()
+        renderMovie(movie)
     }
 
-    fun makeBackButton() {
+    private fun makeBackButton() {
         supportActionBar?.setDisplayHomeAsUpEnabled(true)
     }
 
     private fun renderMovie(
         movie: MovieViewData
     ) {
-        MovieController(
+        MovieController.bind(
             movie = movie,
             MovieView(
                 title = findViewById(R.id.movie_reservation_result_title),
             )
-        ).render()
+        )
     }
 
     private fun renderReservationDetail(
-        reservationDetail: ReservationDetail
+        reservationDetail: ReservationDetailViewData,
+        seats: SeatsViewData,
+        price: PriceViewData
     ) {
         val date = findViewById<TextView>(R.id.movie_reservation_result_date)
         val peopleCount = findViewById<TextView>(R.id.movie_reservation_result_people_count)
-        val price = findViewById<TextView>(R.id.movie_reservation_result_price)
+        val priceText = findViewById<TextView>(R.id.movie_reservation_result_price)
         val dateFormat =
             DateTimeFormatter.ofPattern(date.context.getString(R.string.reservation_datetime_format))
         date.text = dateFormat.format(reservationDetail.date)
 
         peopleCount.text = peopleCount.context.getString(R.string.reservation_people_count)
-            .format(reservationDetail.peopleCount)
+            .format(reservationDetail.peopleCount, formatSeats(seats))
+
+        val formattedPrice = NumberFormat.getNumberInstance(Locale.US).format(price.value)
 
-        val formattedPrice =
-            NumberFormat.getNumberInstance(Locale.US).format(reservationDetail.getTotalPrice())
+        priceText.text =
+            priceText.context.getString(R.string.reservation_price).format(formattedPrice)
+    }
 
-        price.text = price.context.getString(R.string.reservation_price).format(formattedPrice)
+    private fun formatSeats(seats: SeatsViewData): String {
+        return seats.seats.joinToString {
+            getString(R.string.seat_row_column).format(it.rowCharacter, it.column + 1)
+        }
     }
```

Comment:
```suggestion
    private fun formatSeats(seats: SeatsViewData): String {
        return seats.seats.joinToString {
            getString(
                R.string.seat_row_column,
                it.rowCharacter, 
                it.column + 1
            )
        }
    }
```
joinToString을 활용하여 문구를 잘 만들어주셨네요. 👍 
안드로이드에서는 String.format을 사용하지 않고도 format을 활용할 수 있어요.

Reply:
getString()의 format 기능을 활용했습니다!